Sources/ApexyAlamofire/AlamofireClient.swift

```diff
@@ -179,6 +178,36 @@ open class AlamofireClient: Client {
         progress.cancellationHandler = { [weak request] in request?.cancel() }
         return progress
     }
+    
+    @available(macOS 12, iOS 15, watchOS 8, tvOS 15, *)
+    public func request<T>(_ endpoint: T) async throws -> T.Content where T : Endpoint {
+        typealias ContentContinuation = CheckedContinuation<T.Content, Error>
+        return try await withCheckedThrowingContinuation { (continuation: ContentContinuation) in
+            _ = request(endpoint) { result in
```

Comment:
is it possible to add a task cancellation ?

[`withTaskCancellationHandler `](https://developer.apple.com/documentation/swift/3814990-withtaskcancellationhandler)

Reply:
done, but now that looks ugly in View Controller